scripts/filter_fdb_entries.py

```diff
@@ -31,6 +31,8 @@ def get_vlan_cidr_map(filename):
     vlan_cidr = defaultdict()
     if "VLAN_INTERFACE" in config_db_entries.keys() and "VLAN" in config_db_entries.keys():
         for vlan_key in config_db_entries["VLAN_INTERFACE"].keys():
+            if '|' not in vlan_key:
```

Comment:
Can you please use this [method ](https://github.com/Azure/sonic-utilities/blob/master/scripts/db_migrator.py#L64)to check this for consistency?

Reply:
how does this work? the key is a string, how would it be a tuple?

Reply:
well, if it has the ip prefix, it is a tuple.. if not, its a string.

Reply:
has an offline chat with @prsunny regarding this. This does not work here as the referenced call had an underlying API that would convert the key into tuple. In this case on hand, the processing is done within this script.

Reply:
is it better then to check for "/" to check if prefix is present?